### DISCUSSION TITLE
如何将存储从LeanCloud转到Github？

### DISCUSSION THREAD

**Discussion by codejiangqihan** [walinejs/waline#1635] [Q&A] [answered]
LeanCloud的存储和流量已经不够用了，怎么才能在确保数据安全的情况下将存储从LeanCloud转到Github？

**Comment by lizheming** [comment, marked as answer]
后台导出再导入即可。另外如果 LeanCloud 的存储和流量都不够用，不建议使用GitHub存储，会有性能问题，建议使用 postgresql 存储，免费的服务比较多。

**Reply by codejiangqihan** [reply]
Github的速度怎么样？
